### ISSUE TITLE
Age restriction issues

### ISSUE THREAD

**Issue by Rin-Rem** [yuliskov/SmartTubeNext#1077]
So all of a sudden, certain videos at random I'm guessing are age restricted or "offensive".

When I click on these videos all I get is 
"Sign in to confirm your age • This video may be inappropriate for some users"

Thing is, I AM signed in, and my account IS above 18. I'm 25 for fuck sake. This needs to be fixed cause this seems new. And is irritating. God forbid I try to watch any Smi77y videos that get age restricted.

**Comment by Virusmater** [comment]
Yes, some videos like this:
https://www.youtube.com/watch?v=35MQGHm4CfU
aren't possible to watch. On laptop it is possible to use user scripts to avoid this restriction:
https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/

**Comment by costinmx** [comment]
I've solved this issue for my account some weeks ago. You have to verify your age on your **Google** account. This is possible with credit card details (no billing). Never had this issue after verifying age, Your video works fine for me (no restriction) with SmartTube next. It's a YT restriction on your account, STN is just following the rules. Nothing wrong here.

**Comment by Virusmater** [comment]
Yes, I am just annoyed by the fact that I need to proof my age with a credit card
